### PR TITLE
Ver 0.2.4

### PR DESCRIPTION
**Pull request by danielhfrank**
Add put_item method, and fixes bug specifying range key conditions in queries

### CODE REVIEW

File: CHANGELOG [hunk 1]
```diff
@@ -1,3 +1,7 @@
+Version 0.2.4 - 2012-02-023
+    * Fix RangeKeyAttribute bug in queries
+	* Add put_item method
```
Comment (mreiferson) [inline CHANGELOG]: odd formatting?

File: CHANGELOG [hunk 2]
```diff
@@ -1,3 +1,7 @@
+Version 0.2.4 - 2012-02-023
```
Comment (mreiferson) [inline CHANGELOG]: date typo

File: CHANGELOG (hunk 1 above)
Comment (danielhfrank) [inline CHANGELOG]: hard tab. fixed

File: CHANGELOG (hunk 2 above)
Comment (danielhfrank) [inline CHANGELOG]: no, just more precise

**Pull request merged**